test/acceptance/features/bindAppToService.feature

```diff
@@ -528,6 +527,7 @@ Feature: Bind an application to a service
             metadata:
               name: binding-request-etcd
             spec:
+              namingStrategy: "ETCDCLUSTER_{{ .name | upper }}"
```

Comment:
is this still needed?

Reply:
It is required for this specific case where we have `detectBindingResources` true.
In that case auto generated `KIND` is `Service` not `EtcdCluster`. since all owned resources become separate service contexts